docs/source/gbd2019_models/causes/neoplasms/multiple_myeloma/cancer_model.rst

```diff
@@ -131,53 +126,261 @@ Vivarium Modeling Strategy
+and survival regression analysis supported by Flatiron data.
 
 Model Assumptions and Limitations
 +++++++++++++++++++++++++++++++++
 
-.. todo::
-
-   Describe model assumptions and limitations.
-
+1. This cause model assumes no recovery from MM and RRMM since myeloma is an 
+   incurable disease. Patients with MM will inevitably develop relapse and the 
+   health outcomes worsen with every relapse and line of treatment.
+2. This cause model assumes that the GBD incidence rate corresponding to the incidence 
+   of symptomatic MM. That's said, we are comfortable using GBD incidence of MM 
+   as the detection rate of symptomatic MM cases. The incidence of RRMM will be 
+   calculated from survival regression analysis using Cox's proportional hazard model.
+3. The asymptomatic/idolent state (smoldering MM) is exlcuded from this cause 
+   model because we are not interested in the screening and early managment for 
+   MM. As a result, the simulation will not track/model simulants with asymptomatic 
+   condition.
+4. YLDs and YLLs are trivial outcomes for this cause model. Consequently, we 
+   will not build a disability component to capture such outcomes. (TBD)
 
 Cause Model Diagram
 +++++++++++++++++++
 
-.. image:: cause_model_multiple_myeloma.svg
+.. image:: cause_model_diagram.svg
 
 State and Transition Data Tables
 ++++++++++++++++++++++++++++++++
+
 .. list-table:: State Definitions
-   :widths: 1, 10, 15
+   :widths: 1, 5, 15
    :header-rows: 1
 
    * - State
      - State Name
      - Definition
    * - S
-     - **S**\ usceptible
-     - Susceptible to MM
-   * - PC
-     - **P**\ re- **C**\linical, detectable MM cancer
-     - With asymptomatic condition, screen-detectable, will progress to clinical phase
-   * - C
-     - **C**\ linical MM cancer
+     - Susceptible
+     - Susceptible to MM, without condition
+   * - MM
+     - Multiple myeloma
      - With symptomatic condition
-   * - RR
-     - **R**\ elapsed/ **R**\efractory MM
-     - With condition which becomes non-responsive or progressive on therapy or within 60 days of the last treatment in patients who had achieved a minimal response (MR) or better on prior therapy.
-   * - R
-     - **R**\ ecovered
-     - Without condition; not susceptible
+   * - RRMM
+     - Relpased/refractory multiple myeloma
+     - Myeloma returns after initial treatment and can include multiple bouts of 
+       relapse before mortality
+
+.. list-table:: State Data
+   :widths: 1, 5, 15, 15
+   :header-rows: 1
+   
+   * - State
+     - Measure
+     - Value
+     - Notes
+   * - S
+     - prevalence
+     - (1 - prev_c486)
+     - 
+   * - S
+     - excess mortality rate
+     - 0
+     - 
+   * - MM
+     - prevalence
+     - prev_c486 * prevalence ratio of MM to RRMM
+     - The prevalence ratio is based on literature evidence
+   * - MM
+     - excess mortality rate
+     - Derived from survival regression analysis
+     - 
+   * - RRMM
```

Comment:
I was expecting multiple states for 2nd line, 3rd line, 4+ line, instead of one RRMM state

Reply:
Okay, I will divide the RRMM state into multiple stage of relapses.